detekt-rules/src/main/kotlin/OrtImportOrder.kt

```diff
@@ -42,13 +46,31 @@ class OrtImportOrder : Rule() {
     override fun visitImportList(importList: KtImportList) {
         super.visitImportList(importList)
 
-        val importPaths = importList.imports.mapTo(mutableListOf()) { it.importPath.toString() }
-        val sortedImportPaths = importPaths.sorted().toMutableList()
+        /* Need to call importList.node.getChildren(null) instead of importList.getChildren(),
+         * since the latter returns the imports without blank lines.
+         */
+        val children = importList.node.getChildren(null)
+
+        if (children.isEmpty()) return
+
+        val importPaths = children.mapNotNull {
+            when (val psi = it.psi) {
+                is KtImportDirective -> psi.importPath.toString()
+                /* Between two imports there is a child with a psi of type whitespace.
+                 * For 'n' blank lines in between, the text of this child contains
+                 * 'n + 1' line breaks. Thus a single blank line is represented by "\n\n".
+                 */
+                is PsiWhiteSpace -> if (it.text == "\n\n") "" else null
```

Comment:
For consistency, please use the smart-casted `psi` instead of `it` here (unless `psi` really would not work, of course).